lib/saluki-components/src/transforms/aggregate/mod.rs

```diff
@@ -776,6 +776,17 @@ where
                 forwarder.push(Event::Metric(new_metric)).await?;
             }
 
+            if hist_config.copy_to_distribution() {
+                let new_context = context.with_name(format!(
+                    "{}{}",
+                    hist_config.copy_to_distribution_prefix(),
+                    context.name()
+                ));
+
+                let new_metric = Metric::from_parts(new_context, values, metadata.clone());
```

Comment:
We convert the histograms to sketches later [here](https://github.com/DataDog/saluki/blob/4169391f3a0f639320eedeadd7249a5e86a39392/lib/saluki-components/src/destinations/datadog/metrics/request_builder.rs#L614-L622), so I don't think we would need to do any conversion here (?).

Reply:
You're right that we do treat histograms like distributions when sending them out of the Datadog Metrics destination.

I think I'd still want to see us convert to a proper `Distribution` here because the expectation is that we're actively copying a `Histogram` to a `Distribution` by the time it leaves the aggregate transform.

If some other component expected this to happen, but we made this optimization assuming that we only ever fed into the Datadog Metrics destination... well, then we'd have a mismatch in expected behavior.